### PR TITLE
ML2-144 partner copy contentful block

### PR DESCRIPTION
**Pull request by ryan-ludwig**

### CODE REVIEW

File: src/components/CorporateCampaign/CampaignPartner.vue [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+			return this.partnerAreaContent?.contents?.[0]?.headline || '';
+		},
+		bodyCopy() {
+			const richText = this.partnerAreaContent?.contents?.[0]?.bodyCopy;
+			return richText ? documentToHtmlString(richText) : '';
+		},
+	},
+	async mounted() {
+		await this.$nextTick();
+		// make sure all partner content links open externally
```
Comment (ryan-ludwig) [inline src/components/CorporateCampaign/CampaignPartner.vue:58]: Turns out Contentful's field "target" is actually just the URL. I mistook it as a window target. This function should cover it.

File: src/components/CorporateCampaign/CampaignHero.vue [hunk 2]
```diff
@@ -61,19 +61,6 @@ export default {
 		return {};
 	},
 	computed: {
-		heroLogo() {
```
Comment (ryan-ludwig) [inline src/components/CorporateCampaign/CampaignHero.vue:64]: Unused now that the stack of loan cards is there.

File: src/components/CorporateCampaign/CampaignPartner.vue [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+export default {
+	props: {
+		partnerAreaContent: {
+			type: Object,
+			default: () => {},
+		},
+	},
+	computed: {
+		heroLogo() {
```
Comment (mcstover) [inline src/components/CorporateCampaign/CampaignPartner.vue:41]: a little confusing to have this called hero...but no biggie.

File: src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue [hunk 4]
```diff
@@ -449,6 +454,11 @@ export default {
 			const heroArea = contentGroups.campaignHeroCg;
 			return heroArea;
 		},
+		partnerAreaContent() {
+			const contentGroups = this.pageData?.page?.contentGroups;
+			const partnerArea = contentGroups?.partnerAreaCg;
```
Comment (mcstover) [inline src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue:459]: I see now, I need to refactor the contentfulUtil that names these contentGroups... will look at that next week.

**Review by mcstover — APPROVED** [review]

File: src/components/CorporateCampaign/CampaignPartner.vue (hunk 3 above)
Comment (ryan-ludwig) [reply src/components/CorporateCampaign/CampaignPartner.vue:41]: Good catch, agreed

File: src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue (hunk 4 above)
Comment (ryan-ludwig) [reply src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue:459]: I think this is all good. "heroLogo" was just a leftover from a copy/paste.

**Comment by mcstover** [comment]
👍  let's merge and I'll update QA

**Pull request merged**